react/src/pages/SessionLauncherPage.tsx

```diff
@@ -979,6 +999,150 @@ const SessionLauncherPage = () => {
                         </Form.Item>
                       </Flex>
                     </Form.Item>
+                    {supportBatchTimeout ? (
+                      <Form.Item
+                        label={
+                          <Flex gap="xxs">
+                            <Typography.Text>
+                              {t('session.launcher.BatchJobTimeoutDuration')}
+                            </Typography.Text>
+                            <Tooltip
+                              title={t(
+                                'session.launcher.BatchJobTimeoutDurationDesc',
+                              )}
+                            >
+                              <QuestionCircleOutlined />
+                            </Tooltip>
+                          </Flex>
+                        }
+                        extra={
+                          <Form.Item
+                            noStyle
+                            shouldUpdate={(prev, next) =>
+                              prev.batch.timeout !== next.batch.timeout ||
+                              prev.batch.timeoutUnit !== next.batch.timeoutUnit
+                            }
```

Comment:
If `extra` is a React element, `Form.Item` creates a bottom space for it, which causes unnecessary space like this:

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/XqC2uNFuj0wg8I60sMUh/c2559aeb-b2a0-418e-9028-258ac51ea406.png)

We can change the hierarchy as shown below:
```jsx
<Form.Item shouldUpdate>
  <BAIIntervalView render={(text) => <Form.Item label="" extra={text} />}